microscopy_SPIM001/participants.tsv

```diff
@@ -0,0 +1,3 @@
+participant_id	sex	species	strain	strain_rrid
```

Comment:
In readme, we say `rat`. We could change `species` for `rattus norvegicus` and remove `strain` and `strain_rrid` columns.
If we remove columns, they must be removed from the json as well.